index.html

```diff
@@ -0,0 +1,302 @@
+<!doctype html>
+<html lang="ru">
+<head>
+    <meta charset="utf-8"/>
+    <title>Сайт</title>
+    <link rel="stylesheet" href="./css/main.css" />
+</head>
+<body>
+    <main>
+        <header>
+            <nav>
+                <ul class="upper-nav-bar">
+                    <li><a href="#">
+                        Главная
+                    </a></li>
+                    <li><a href="#">
+                        Наши проекты
+                    </a></li>
+                    <li><a href="#">
+                        Услуги
+                    </a></li>
+                    <li><a href="#">
+                        Новости
+                    </a></li>
+                    <li><a href="#">
+                        Контакты
+                    </a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <div class="first-screen">
+            <div>
```

Comment:
Too many `div`s without classes.

Reply:
There are lesser `div`s without classes in code now.